ulog/log.go

```diff
@@ -99,14 +107,59 @@ func TestingLogger() (Log, *spy.Sink) {
 	}, sink
 }
 
-// Logger returns the package-level logger configured for the service
-// TODO:(at) Remove Logger() call, _std and defaultLogger() access in ulog
-func Logger() Log {
+func logger() Log {
+	_setupMu.Lock()
+	defer _setupMu.Unlock()
+
 	return &baseLogger{
 		log: _std.(*baseLogger).log,
```

Comment:
Probably we should have _std to be baseLogger type, to catch a panic during SetLogger..